### DISCUSSION TITLE
Phosphoproteomics on a single raw file

### DISCUSSION THREAD

**Discussion by iamagerm** [vdemichev/DiaNN#776] [Q&A] [answered]
Hello, I have a question on phosphoproteomics analysis with DIA-NN. 

I want to search for phosphorylated proteins in a single sample. My set up is in the image that I attached. Is that correct? 
I'm going to use the default parameters, the only thing i changed is that i ticked the Phospho box and automatically the maximum number of variable modifications went up to 1 (was 0 before i checked the Phospho box). 

On another topic, what is the meaning of this number, why is 0 by default when the boxes for Nterm M excision and C carbamido are checked?

Thanks, I'm new to DIA and DIA-NN.

**Comment by vdemichev** [comment, marked as answer]
In general all OK, but normally you'd want 2 or 3 var mods for phospho and please also see https://github.com/vdemichev/DiaNN#changing-default-settings

**Comment by vdemichev** [comment]
Also, better to generate a predicted library in a separate pipeline step.

**Reply by iamagerm** [reply]
Thanks a lot.
